src/khoj/interface/web/login.html

```diff
@@ -0,0 +1,197 @@
+            color: #475569;
+            font-family: roboto, karma, segoe ui, sans-serif;
+            font-size: 20px;
+            font-weight: 300;
+            line-height: 1.5em;
+        }
+        body > * {
+            padding: 10px;
+            margin: 10px;
+        }
+
+        @keyframes gradient {
+            0% {
+                background-position: 0% 50%;
+            }
+            50% {
+                background-position: 100% 50%;
+            }
+            100% {
+                background-position: 0% 50%;
+            }
+        }
+
+        a.khoj-logo {
+            text-align: center;
+        }
+
+        button#khoj-banner-submit,
+        input#khoj-banner-email {
+            padding: 10px;
+            border-radius: 5px;
+            border: 1px solid #475569;
+            background: #f9fafc;
+        }
+
+        button#khoj-banner-submit:hover,
+        input#khoj-banner-email:hover {
+            box-shadow: 0 0 11px #aaa;
+        }
+
+        div#login-modal {
+            display: grid;
+            grid-template-columns: 1fr;
+            grid-template-rows: 1fr auto auto auto;
+            gap: 10px;
+            padding: 10px;
+            margin: 10px;
+            background: #fff;
+            border-radius: 5px;
+            border: 1px solid #475569;
+            box-shadow: 0 0 11px #aaa;
+            margin-left: 25%;
+            margin-right: 25%;
+        }
+
+        div.g_id_signin {
+            margin: 0 auto;
+            display: block;
+        }
+
+        h1.login-modal-title {
+            text-align: center;
+            line-height: 28px;
+            font-size: x-large;
+        }
+
+        @media only screen and (max-width: 600px) {
+            a.khoj-banner {
+                display: block;
+            }
+            p.khoj-banner {
+                padding: 0;
+            }
+            div#login-modal {
+                margin-left: 10%;
+                margin-right: 10%;
+            }
+        }
+
+    </style>
+    <script>
+        var khojBannerSubmit = document.getElementById("khoj-banner-submit");
+        khojBannerSubmit?.addEventListener("click", function(event) {
+            event.preventDefault();
+            var email = document.getElementById("khoj-banner-email").value;
+            fetch("https://app.khoj.dev/beta/users/", {
+                    method: "POST",
+                    body: JSON.stringify({
+                        email: email
+                    }),
+                    headers: {
+                        "Content-Type": "application/json"
+                    }
+                }).then(function(response) {
+                    return response.json();
+                }).then(function(data) {
+                    console.log(data);
+                    if (data.user != null) {
+                        document.getElementById("khoj-banner").innerHTML = "Thanks for signing up. We'll be in touch soon! 🚀";
```

Comment:
Is this the text to be shown post signup?

Reply:
Oh, that's only if it's a demo instance. We actually don't need this here -- Probably won't have a demo instance + sign-in.